lib/utils/string_extension.dart

```diff
@@ -373,4 +373,24 @@ extension StringCasingExtension on String {
     final match = regex.firstMatch(this);
     return match?.group(1);
   }
+
+  bool isRoomAlias() {
+    final regExp = RegExp(r'^[#][^:]+:.+$');
```

Comment:
same as https://github.com/linagora/twake-on-matrix/pull/1775/files#r1608444829

Reply:
should we write a test ?

Reply:
updated